runbotics/runbotics-orchestrator-ui/src/src-landing/components/MarketplaceCard/MarketplaceCard.tsx

```diff
@@ -47,8 +47,8 @@ const MarketplaceCard: FC<MarketplaceCardProps> = ({ offer, className }) => {
                     <div className={styles.content}>
                         <div className={styles.info}>
                             {offer.tags?.items.length > 0 &&
-                                offer.tags.items.slice(0, 3).map((tag) => (
-                                    <InformationalTag text={tag.name} key={tag.slug}/>
+                                offer.tags.items.filter(tag => tag).slice(0, 3).map((tag) => (
+                                    <InformationalTag text={tag?.name} key={tag?.slug}/>
```

Comment:
I've commented here too I think. have you resolved it? 
it's risky to allow for tag?.slug as a key property which can cause DOM issues, so instead please use 
`.map(tag => tag?.slug && tag?.name ? <InformationalTag text={tag.name} key={tag.slug} /> : null`